zulip/zulip/__init__.py

```diff
@@ -267,12 +267,25 @@ def get_default_config_filename():
     config_file = os.path.join(os.environ["HOME"], ".zuliprc")
     if (not os.path.exists(config_file) and
             os.path.exists(os.path.join(os.environ["HOME"], ".humbugrc"))):
-        raise RuntimeError("The Zulip API configuration file is now ~/.zuliprc; please run:\n\n"
+        raise ZulipError("The Zulip API configuration file is now ~/.zuliprc; please run:\n\n"
                            "  mv ~/.humbugrc ~/.zuliprc\n")
     return config_file
 
 class ZulipError(Exception):
-    pass
+    def __init__(self,msg):
```

Comment:
You don't need constructors for a new exception: If it inherits from Exception, the message output is handled automatically.